src/SignhostAPIClient/Rest/DataObjects/Transaction.cs

```diff
@@ -7,9 +7,11 @@ public class Transaction
 	{
 		public string Id { get; set; }
 
+		public IDictionary<string, FileEntry> Files { get; set; }
```

Comment:
Please discuss: This is a readonly property in the API. Should we comment these kind of properties, or should we place them in a TransactionResponse object which inherits the Transaction object?